src/css/aclmanagement.css

```diff
@@ -44,7 +44,7 @@
 }
 
 .acl-management-view .acl-rules .index-cell {
-    padding: 0;
+    padding: 0.8rem;
```

Comment:
This is not OK. It will make the text in the index column overflow out of the cell

Reply:
fixed